### PR TITLE
Added custom timer arg for testing

### PR DESCRIPTION
**Pull request by JakeCooper**

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -31,12 +33,26 @@ type quoteResponse struct {
 	cyrptokey string
 }
 
+var (
+	delayVal = rand.Intn(4) + 1
```
Comment (trstephen) [inline server.go:37]: `delayVal` is now a constant, instead of changing every time a response is sent

File: server.go [hunk 2]
```diff
@@ -10,6 +10,8 @@ import (
 	"os"
 	"strings"
 	"time"
+	"flag"
```
Comment (trstephen) [inline server.go]: Nit: Should be alpha sorted

(`gometalinter` will complain the next time I touch the file 😜 )

File: server.go [hunk 3]
```diff
@@ -31,12 +33,26 @@ type quoteResponse struct {
...
+
 func main() {
 	// If we don't provide a seed for rand then it hehaves as if
 	// we ran Seed(1). It's not safe to run this in concurrent
 	// code so I'm doing it here!
+	flag.Parse()
 	rand.Seed(time.Now().Unix())
 
+	if len(flag.Args()) != 0 {
+		parsedVal, err := strconv.Atoi(flag.Args()[0])
```
Comment (trstephen) [inline server.go:48]: Nit: Could use `flag.Int` or `flag.VarInt` to handle this when `flag.Parse()` is called.

File: server.go [hunk 4]
```diff
@@ -57,9 +73,10 @@ func main() {
 
 		//logs an incoming message
 		fmt.Printf("Received message %s -> %s \n", conn.RemoteAddr(), conn.LocalAddr())
-
+		fmt.Printf("Waiting %d seconds\n", delayVal)
```
Comment (trstephen) [inline server.go]: 😂

**Review by trstephen — CHANGES_REQUESTED** [review]
Need to retain default random behavior

File: server.go (hunk 1 above)
Comment (JakeCooper) [reply server.go:37]: Which is alright just for basic quicktests. Obviously you should use the regular random when fulltesting.

File: server.go (hunk 2 above)
Comment (JakeCooper) [reply server.go]: :+1:

File: server.go (hunk 1 above)
Comment (trstephen) [reply server.go:37]: Yes, but individual response times are no longer random because `delayVal` never changes after it's called the first time.

Comment (JakeCooper) [reply server.go:37]: Aww shiet you're right.

Comment (JakeCooper) [reply server.go:37]: Resolved

File: server.go (hunk 4 above)
Comment (trstephen) [reply server.go]: This logs the **last** `delayVal`. It should go at L110 to capture the updated value that will be used for the wait.

**Comment by trstephen** [comment]
LGTM once logging statement is moved.

**Review by trstephen — APPROVED** [review]

**Pull request merged**